plugin/src/test/sc/plugin2021/PieceTest.kt

```diff
@@ -40,14 +42,22 @@ class PieceTest: WordSpec({
             }
         }
         "converted to String" Should {
```

Comment:
If you really prefer `should` over `Should` then please be consistent

Reply:
hah right, missed that ^^